Gemfile

```
@@ -49,7 +49,7 @@ gem 'avalon-about', git: 'https://github.com/avalonmediasystem/avalon-about.git'
 #gem 'bootstrap-sass', '< 3.4.1' # Pin to less than 3.4.1 due to change in behavior with popovers
 gem 'bootstrap-toggle-rails'
 gem 'bootstrap_form'
-gem 'iiif_manifest', '~> 0.6'
+gem 'iiif_manifest', git: "https://github.com/samvera/iiif_manifest.git", branch: 'main'
```

`iiif_manifest` 1.3.0 was released last week which should include your fixes.
```suggestion
gem 'iiif_manifest', '~> 1.3'
```